### PR TITLE
Rearrange savani_perturbation to be the same as s40rts_perturbation

### PR DESCRIPTION
**Pull request by SophieCoulson**
Here I've simply rearranged the savani_perturbation plug-in so there is a function to return Vs, and this can be called from another plug in. It is now in exactly the same format as s40rts_perturbation.

### CODE REVIEW

File: source/initial_temperature/SAVANI_perturbation.cc [hunk 1]
```diff
@@ -188,101 +189,120 @@ namespace aspect
       return 0;
     }
 
+    template <>
+    double
+	SAVANIPerturbation<2>::
+    get_Vs (const Point<2> &/*position*/) const
```
Comment (bangerth) [inline source/initial_temperature/SAVANI_perturbation.cc]: You'll have to run the indent script.

**Comment by SophieCoulson** [comment]
Done

File: source/initial_temperature/SAVANI_perturbation.cc [hunk 2]
```diff
@@ -176,6 +176,7 @@ namespace aspect
 
     }
 
+
```
Comment (gassmoeller) [inline source/initial_temperature/SAVANI_perturbation.cc]: while you are at it, add another empty line. We usually have 1 empty line between functions in header files, and 3 empty lines between functions in .cc files.

Comment (gassmoeller) [reply source/initial_temperature/SAVANI_perturbation.cc]: Thinking about this again, it is really not necessary to hold up this PR for an empty line that was missing before already. Ready to merge.

**Pull request merged**